backend/src/main/java/ai/verta/modeldb/versioning/RepositoryDAORdbImpl.java

```diff
@@ -544,8 +549,9 @@ public Boolean deleteRepositories(List<String> repositoryIds, ExperimentRunDAO e
       ListRepositoriesRequest.Response.Builder builder =
           ListRepositoriesRequest.Response.newBuilder();
 
-      repositoryEntities.forEach(
-          repositoryEntity -> builder.addRepositories(repositoryEntity.toProto()));
+      for (RepositoryEntity repositoryEntity : repositoryEntities) {
```

Comment:
why was this necessary?

Reply:
While converting Repository object to proto object it will throw `InvalidProtocolBufferException` exception dure to `keyvalue` conversion and we can not rethrow it from lambda so I have to change it to normal foreach.